pages/compare.py

```diff
@@ -493,97 +505,98 @@ def render_card(container, input, caller_id, completions):
     )
     container.code(d['completion'], language=None)
 
+if st.button("Generate Completions"):
```

Comment:
when there is a button is there any issue getting the other recent completions from users? I feel like I've run into that issue before when having multiple sections with a button half way through.

Reply:
confirmed